kubernetes/zenko/charts/backbeat/values.yaml

```diff
@@ -9,8 +9,8 @@ global:
   replicationEndpoints: []
 
 image:
-  repository: zenko/backbeat
-  tag: 8.2.4
+  repository: registry.scality.com/backbeat/backbeat
```

Comment:
This wouldn't work for our installs as they dont have access to registry.scality.com, I can help you push it to dockerhub